src/common/meta/src/ddl_manager.rs

```diff
@@ -418,6 +415,75 @@ impl DdlManager {
 
         Ok((procedure_id, output))
     }
+
+    pub async fn submit_ddl_task(
+        &self,
+        ctx: &ExecutorContext,
+        request: SubmitDdlTaskRequest,
+    ) -> Result<SubmitDdlTaskResponse> {
+        let span = ctx
+            .tracing_context
+            .as_ref()
+            .map(TracingContext::from_w3c)
+            .unwrap_or(TracingContext::from_current_span())
```

Comment:
```suggestion
            .unwrap_or_else(|| TracingContext::from_current_span())
```

Reply:
resolved in [7ba42fe](https://github.com/GreptimeTeam/greptimedb/pull/6625/commits/7ba42fe08c13de535dc8f49ffa8d70a097fccb36)